tests/AllBinding/AllBinding.log

```diff
@@ -266,9 +271,36 @@ design: (work@top)
       |vpiActual:
       \_enum_var: (work@top.dmem2exu_resp_i), line:1:57, endln:1:72
     |vpiTypedef:
-    \_enum_typespec: (type_scr1_mem_resp_e), line:1:1, endln:5:24
+    \_ref_obj: (work@top.dmem2exu_resp_i)
+      |vpiParent:
+      \_port: (dmem2exu_resp_i), line:1:57, endln:1:72
+      |vpiFullName:work@top.dmem2exu_resp_i
+      |vpiActual:
+      \_enum_typespec: (type_scr1_mem_resp_e), line:1:1, endln:5:24
     |vpiInstance:
     \_module_inst: work@top (work@top), file:${SURELOG_DIR}/tests/AllBinding/dut.sv, line:1:1, endln:5:10
+\_weaklyReferenced:
```

Comment:
What is this new relation? That is not in the Standard Object Model. That is an unwanted departure from the Standard.
I can't tell what this means.

Reply:
Please don't make this:

"A near future change will collect the typespecs under a scope and populate the scope::typespecs vector and show these entries under the weakReferences section will move under there and this will disappear."

an other backward compatibility problem with existing code. Existing UHDM code need to be able to navigate the typespecs from their logical anchor points.